de.fu_berlin.inf.dpp.whiteboard/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -55,6 +60,10 @@ eclipse.preferences.version=1
 editor_save_participant_org.eclipse.jdt.ui.postsavelistener.cleanup=true
 formatter_profile=_Saros Formatter
 formatter_settings_version=12
+org.eclipse.jdt.ui.ignorelowercasenames=true
+org.eclipse.jdt.ui.importorder=\#;;
```

Comment:
does any one have any idea what does `\#;` stand for?

Reply:
I don't really get the syntax, but it should be the setting that sets the import order to match the google style:
```
<static import block>

<import block>
```

But it doesn't really matter as it was automatically generated by eclipse.